pkg/ovs/openflow/interfaces.go

```diff
@@ -100,6 +101,7 @@ type Bridge interface {
 	CreateTable(table Table, next uint8, missAction MissActionType) Table
 	// AddTable adds table on the Bridge. Return true if the operation succeeds, otherwise return false.
```

Comment:
I suppose this is a stale comment which can be removed?

Reply:
Yes, removed.